test/lit/cast-to-basic.wast

```diff
@@ -18,12 +18,14 @@
 
   ;; CHECK:      (func $cast (type $none_=>_none)
   ;; CHECK-NEXT:  (drop
-  ;; CHECK-NEXT:   (ref.cast null struct
+  ;; CHECK-NEXT:   (ref.cast null none
   ;; CHECK-NEXT:    (ref.null none)
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
   (func $cast
+    ;; Note that this will not round-trip precisely because Binaryen IR will
```

Comment:
Since this test is explicitly testing round-tripping, it might be good to update this test case to use a `(ref null struct)` parameter instead.